src/components/shared/Input.js

```diff
@@ -7,10 +7,31 @@ import { Context } from '../../utils/context'
 const PasswordInput = styled.input((props) => props)
 const Label = styled.label((props) => props)
 const Button = styled.button((props) => props)
+const Warning = styled.p((props) => props)
 
 export default function Input(demo1) {
   const { update } = React.useContext(Context)
   const { password } = demo1.theme
+  const [passwordMinError, SetPasswordMinError] = useState(false)
+  const [passwordMaxError, SetPasswordMaxError] = useState(false)
+  const passwordLowerBound = 1
```

Comment:
I don't think we need `passwordLowerBound` and the naming here is just confusing. Can we not just check for `lenght > 0`?